### ISSUE TITLE
DV alone with CLI - docker run command missing flags

### ISSUE THREAD

**Issue by KaloyanTanev** [ObolNetwork/obol-docs#342]
# 🎯 Problem to be solved

While trying to launch a cluster alone following the CLI in the docs, it didn't seem as streamlined as I hoped for.

In [step 1.2](https://docs.obol.tech/docs/start/quickstart_alone#step-1-create-the-key-shares-locally) there is code snippet that I don't quite get:
```
WITHDRAWAL_ADDR=[ENTER YOUR WITHDRAWAL ADDRESS HERE]
FEE_RECIPIENT_ADDR=[ENTER YOUR FEE RECIPIENT ADDRESS HERE]
NETWORK="holesky"
```
Those values are not exported, neither do they exist in the `.env`.

Furthermore in step 1.4 the command:
```
docker run --rm -v "$(pwd):/opt/charon" --env-file .env obolnetwork/charon:v0.19.0 create cluster
```
returns:
```
20:43:32.172 ERRO cmd        Fatal error: missing --nodes flag       
	cmd/createcluster.go:310 .validateCreateConfig
	cmd/createcluster.go:145 .runCreateCluster
	cmd/createcluster.go:91 .func1
	cmd/cmd.go:80 .func1
	main.go:19 .main
```
while I do understand (after diving into the charon code) that there should be either `--nodes` flag or `--definition-file`, from where they are fetched, I believe it should be a bit more clear for the user what to do, with the example code snippets running out of the box.

# 🛠️ Proposed solution

1. Remove
```
WITHDRAWAL_ADDR=[ENTER YOUR WITHDRAWAL ADDRESS HERE]
FEE_RECIPIENT_ADDR=[ENTER YOUR FEE RECIPIENT ADDRESS HERE]
NETWORK="holesky"
```
if it is not required.

2. Add the necessary flags to the `docker run --rm -v "$(pwd):/opt/charon" --env-file .env obolnetwork/charon:v0.19.0 create cluster` command so the user can see what is the bare minimum config required to run. Potentially again with similar `[ENTER YOUR WITHDRAWAL ADDRESS HERE]` values where necessary.

Or

Add example definition file that the user can use and pass to the `docker run --rm -v "$(pwd):/opt/charon" --env-file .env obolnetwork/charon:v0.19.0 create cluster` command. Potentially again with similar `[ENTER YOUR WITHDRAWAL ADDRESS HERE]` values where necessary.

# 🧪 Tests

Be able to seamlessly follow the instructions 1:1 for running cluster alone via the CLI.

- [ ] Netlify deployment build tests passing

**Comment by slugmann321** [comment]
This is a bit confusing... it seems that you started on the "launchpad" flow of the docs, but then switched to the CLI flow. 

The "git clone" part is missing from the launchpad tab. This is being fixed with PR #343. Other than that, the launchpad flow works fine. 

For the CLI flow, it is true that we should be more clear that WITHDRAWAL_ADDR and FEE_RECIPIENT_ADDR are not found in the .env file but rather they are flags that could be added to the "docker run" command. I agree that we should present the docker run command with these flags. 

Will implement in PR #343

**Comment by slugmann321** [comment]
Ok I fixed the CLI flow for "create a cluster alone" so that it's clear. Added the mandatory flags so that the command can be run easily. 

See https://github.com/ObolNetwork/obol-docs/pull/343
